core/src/main/java/org/elasticsearch/search/aggregations/pipeline/PipelineAggregatorFactory.java

```diff
@@ -158,6 +158,10 @@ public final XContentBuilder toXContent(XContentBuilder builder, Params params)
         return builder.endObject();
     }
 
+    protected boolean overrideBucketsPath() {
```

Comment:
Will do. This is actually the same change as made in https://github.com/elastic/elasticsearch/pull/15014 where I have already added JavaDocs. I will make sure that one is merged first and the JavaDoc is retained